lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -79,7 +79,7 @@ def execute_all(event, object)
       # This subscription was re-evaluated.
       # Send it to the specific stream where this client was waiting.
       def deliver(subscription_id, result)
-        payload = { result: result.to_h, more: true }
+        payload = {result: result.to_h, more: true}
```

Comment:
Those are all whitespace changes that happened on save. May drop them if you don't want them, but they should conform to the repo's rubocop config.